Makefile

```diff
@@ -123,6 +123,13 @@ project-aws-get-cognito-client-secret: # Get AWS secret - mandatory: NAME
 		--query 'UserPoolClient.ClientSecret' \
 		--output text
 
+project-aws-get-admin-secret: #Get AWS Pass
+	aws secretsmanager get-secret-value \
+		--secret-id $(PROJECT_GROUP_SHORT)-$(PROJECT_NAME_SHORT)-$(ENVIRONMENT)-cognito-admin-password \
+		--region $(AWS_REGION) \
+		--query 'SecretString' \
+		--output text
+
```

Comment:
Talk to @daonhs about who is going to merge this in first. One of you guys will need to update these details once the new cognito users have been added

Also double check with @jope1 as to where we should be getting the cognito passwords for as we wont be using the cognito admin password anymore